src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs

```diff
@@ -30,9 +30,104 @@ public FeatureManagementKeyValueAdapter(FeatureFilterTracing featureFilterTracin
 
             var keyValues = new List<KeyValuePair<string, string>>();
 
+            // Check if we need to process the feature flag using the microsoft schema
+            if (featureFlag.Variants != null || featureFlag.Allocation != null || featureFlag.Telemetry != null)
```

Comment:
I'd like to treat null and empty the same. Similarly to how client_filters are treated.